pages/scf_join.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: page-fullwidth
+permalink: /scf/join/
+title: Become a Partner
+---
+![Stockholm 2014]({{site.filesurl}}/workshops/scilifelab-2014-05-slice.jpg)
+
+A Partner is an organization that has made a commitment to the growth and spread of Software Carpentry. Partners receive annual access to instructor training and guidance with capacity building at their organization. Partnerships are able to run low-cost local workshops while developing communities of practice around effective data-driven reproducible research. Community developed lessons enable our instructors to globally share the effort for developing impactful lessons on technical topics. Partnerships give Software and Data Carpentry revenue to ensure the ongoing development and maintenance of the lessons demanded by research communities. We work to give your local instructors support, mentorship and a forum for community lesson development.
+
+We offer several levels of Partnerships, based on training activities expected annually.
+
+### Tiers
+
+
+
+||Bronze|Silver|Gold|Platinum|
+| ------|:------:|:------:|:------:|:------:|
+|# of Coordinated Workshops|2|4|6|negotiable|
+|Discount for additional <br>coordinated workshops|20%|33%|50%|negotiable|
+|Self-organized workshops<br> at partner organization **|no-charge|no-charge|no-charge|no-charge|
+|Number of instructors <br>trained ***|0|6 online|15 with possibility <br>for in-person^<br>training event|negotiable|
+|Seat on the <br>SCF Advisory Board|No|Yes|Yes|Yes|
+|Train an in-house instructor <br>trainer at partner org|No|No|No|Available|
+|Lesson development <br>services|No|No|No|Available|
+|Joint Fee (annual)|[$5,000](mailto:partnerships@software-carpentry.org?subject=Interested in Bronze Partnership)|[$7,500](mailto:partnerships@software-carpentry.org?subject=Interested in Silver Partnership)|[$15,000](mailto:partnerships@software-carpentry.org?subject=Interested in Gold Partnership)|[Contact us](mailto:partnerships@software-carpentry.org?subject=Interested in Platinum Partnership)|
+|SC or DC Only (annual)|$4,000|$6,500|$12,500|Contact us|
+
+
+
+\* For Coordinated workshops, we will find instructors for the workshop, set up and handle registration, support instructor logistics and coordinate and share assessment results with the Partner organization. A designated contact approves coordinated workshop requests for that organization. Instructor travel expenses are the responsibility of the partner and are not covered in Partnership fees.
```

Comment:
Please insert section heading with anchor here so that others can link directly to this explanation of coordinated vs. self-organized workshops (e.g., from instructor training).